F/FLTK/build_tarballs.jl

```diff
@@ -0,0 +1,79 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "FLTK"
+version = v"1.3.8"
+
+# Collection of sources required to build FLTK
+sources = [
+    ArchiveSource("https://www.fltk.org/pub/fltk/$(version)/fltk-$(version)-source.tar.gz",
+                  "f3c1102b07eb0e7a50538f9fc9037c18387165bc70d4b626e94ab725b9d4d1bf")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add fltk fltk-dev fltk-fluid # Host FLUID install required for cross-compilation
+if [[ ${target} == x86_64-linux-musl* ]]; then
+    # Remove host system libraries otherwise get "undefined reference to getrandom" error message on x86_64
+    rm /usr/lib/libexpat*
+    rm /lib/libuuid*
+    rm /usr/lib/libfontconfig*
+fi
+cd ${WORKSPACE}/srcdir/fltk-*
+mkdir build
+cd build
+cmake .. \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DOPTION_BUILD_SHARED_LIBS=ON \
+    -DOPTION_USE_SYSTEM_LIBJPEG=ON \
+    -DOPTION_USE_SYSTEM_LIBPNG=ON \
+    -DOPTION_USE_SYSTEM_ZLIB=ON \
+    -DOPTION_USE_THREADS=ON
+make -j${nproc}
+make install
+if [[ ${target} == *mingw* ]]; then
+    # Rename binaries to correct extension
+    mv ${bindir}/fltk-config ${bindir}/fltk-config${exeext}
+fi
```

Comment:
I'm not sure it's important to have fltk-config listed in the products

Reply:
I agree, removed in b12f87e.